Libs/Common/Shapeworks.h

```diff
@@ -35,16 +40,26 @@ using IdentityTransform  = itk::IdentityTransform<double, 3>;
 using TransformPtr       = GenericTransform::Pointer;
 TransformPtr createTransform(const Matrix33 &mat, const Vector3 &translate = makeVector({0,0,0}));
 
+/// make a plane
+Plane makePlane(const Vector3 &n, const Point &o);
+
 /// Affine transforms are used for many Image manipulation commands
 using AffineTransform    = itk::AffineTransform<double, 3>;
 using AffineTransformPtr = AffineTransform::Pointer;
 
+/// Mesh transforms
+using vtkTransform = vtkSmartPointer<vtkTransform>;
```

Comment:
I think this is extremely confusing.  It also breaks a bunch of things in the studio_mesh_reconstruction branch now that it's including this file.  You can see in #927 how I fixed it.

Reply:
You used swTransform in place of vtkTransform, correct?

Reply:
Yes, I had to rename it since Studio uses these VTK classes and expects them to not have been overridden to something else.

Reply:
Okay. Pushed a commit to change it from vtkTransform to swTransform.

Reply:
You don't need to do that unless we are not planning to merge #927

Reply:
In fact, now I have a bunch of conflicts to deal with.